src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -5,13 +5,18 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Numerics.Hashing;
+using System.ComponentModel;
 
 namespace System.Drawing
 {
     [DebuggerDisplay("{NameAndARGBValue}")]
     [Serializable]
 #if !MONO
     [System.Runtime.CompilerServices.TypeForwardedFrom("System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+#endif
+#if !MONOTOUCH && !MONOMAC && FEATURE_TYPECONVERTER
```

Comment:
we're not defining`MONOMAC` in Mono for XM but I see you just copied it from the existing Color.cs so it's fine, I'll file an issue to see what we should replace it with.

Also `FEATURE_TYPECONVERTER` isn't set on any of the mobile profiles so `!MONOTOUCH && !MONOMAC` is redundant. Maybe we can just use that.

Reply:
Done, thanks for noticing